.github/jobs/ios.yml

```diff
@@ -1,14 +1,20 @@
+parameters:
+  name: ''
+  vmImage: ''
+  xCodeVersion: ''
+  simulator: ''
```

Comment:
Nit: With these parameters, it might be a good idea to format things closer to the [schema definition](https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/parameters-parameter?view=azure-pipelines) for parameters:
```
# Example
parameters:
- name: name
   type: string
   default: ''
```
Also, since `name` is a property of a parameter, it might help to rename it to something like `jobName`.